新規登録画面(signup-page)の製造開始

hi-lee-mon opened the pull request:
### 期待されること（できるだけ細かく編集・追加すること）

<!-- レビューをする人が何を確認すれ良いのかがわかるように書く -->

- [x]  以下すべての要素を囲む丸型のカード
- [x]  丸型のアイコン
- [x]  丸アイコンのすぐ下に"写真を登録"リンク
- [x]  "写真を登録"リンクは「青色」※Figmaテキストカラーの項目参照
- [x]  (A) placeholder="Name"のテキストフィールド
- [x]  (B) placeholder="Email"のテキストフィールド
- [x]  (C)placeholder="Password"のテキストフィールド
- [x]  (A) (B) (C) の3つのテキストフィールドは縦に並べてくっつけ、丸型にする
- [x]  "登録"が描画されるボタン
- [x]  "登録"ボタンは「青色」※Figmaテキストカラーの項目に無いため暫定でLinkの色を使用
- [x]  ログイン画面へ遷移するためのリンクを追加（追加作成後にデザインナーに確認）
- [x]  "登録"ボタン押下でHomeへ遷移

【補足事項】

### UI（スクリーンショット）
![image](https://user-images.githubusercontent.com/79642192/141666274-79bab6d9-8602-4bcc-945f-b8e98ec86ac8.png)

<!-- 見た目を変更した場合のみ -->

### 懸念点

<!-- 作業における不安や内容の確認があれば書く -->

- "登録"ボタンの色が定義されていない。Linkの色を使用。
- "登録"ボタンの色は先を使用。#2c7a7b

### 最終確認事項

<!-- 確認した問題点は可能な限り対応し,難しい場合は懸念点やその他に記載する -->

- [x]  動作確認をしたか？
- [x]  新たにエラーは発生していないか？
- [x]  merge後に新たなタスクやIssueは発生するか？
- [x]  これによる影響範囲を考え、それらに修正がある場合はそれをしたか？

### その他

-

hi-lee-mon committed:
start work

hi-lee-mon committed:
新規登録画面(signup-page)の製造完了

nbr41to commented on public/images/icon 1.svg:
```diff
@@ -0,0 +1,9 @@
+<svg width="118" height="121" viewBox="0 0 118 121" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
こちら`src/assets/svg`に保存するようにお願いします🙇‍♂️

nbr41to reviewed (commented):
一点コメントしました！

hi-lee-mon committed:
第1回レビュー後修正
・svgファイルをassets/svg配下に移動
・縦幅をheight:auto;で指定するよう修正
⇒縦幅を小さくすると画面崩れが起きていたためauto指定に修正

hi-lee-mon commented:
レビューありがとうございます！
svgファイルは`src/assets/svg`に配置するようにしました。

nbr41to reviewed (approved):
こちらもLGTMです！
Conflictだけ直したらmergeお願いします！

nonn1525 reviewed (approved):
LGTM!

hi-lee-mon committed:
Merge branch 'develop' into feature/signup-page

hi-lee-mon committed:
コンフリクト解消

hi-lee-mon committed:
signup-pageに不要なコードがあるため削除
